wolfcrypt/test/test.c

```diff
@@ -11844,6 +11844,7 @@ int rsa_test(void)
 #ifndef WOLFSSL_RSA_VERIFY_INLINE
 
 #if defined(WOLFSSL_CRYPTOCELL)
+/* Cryptocell requires the input data and signature to verify */
```

Comment:
Can you clarify this comment? It sounds like the Cryptocell core expects different input parameters, but its not clear to me from this comment what that is. Perhaps indent this comment?

Reply:
Done